features/step_definitions/networking.rb

```diff
@@ -1253,3 +1253,18 @@
   end
   logger.info "The egressfirewall type is stored to the #{cb_name} clipboard."
 end
+
+Given /^I save ipecho url to the#{OPT_SYM} clipboard$/ do | cb_name |
+  ensure_admin_tagged
+  cb_name = "cb_ipecho_url" unless cb_name
+  cb[cb_name]="172.31.249.80:9095"
```

Comment:
maybe we should add a comment about who owns this service, so we know who to contact?  Or is it documented somewhere else?

Reply:
@rbbratta Thanks Ross! The ipecho  is running on a VM in vmc network, and currently maintained by me.  :)

Reply:
@huiran0826 , this `ipecho` service needs some description of what it is and what is it used for. Doing a quick search I can't for sure know what this should mean here.

Reply:
@akostadinov This ipecho service is used for egress ip feature testing.  It echos the source IP of the outbound traffic from cluster.  I will add comments about this step in next PR.  Thanks!